### PR TITLE
Add migaloo network config

### PR DESCRIPTION
**Pull request by kakucodes**
First attempt at adding Migaloo chain.
https://docs.migaloo.zone/validators/testnet

### CODE REVIEW

File: cw-orch/src/daemon/networks/migaloo.rs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    chain_id: "migaloo-chain",
+    gas_denom: "uwhale",
+    gas_price: 0.1,
+    grpc_urls: &["http://localhost:9090"],
+    network_info: MIGALOO_NETWORK,
+    lcd_url: None,
+    fcd_url: None,
+};
+
+pub const NARWHAL_2: ChainInfo = ChainInfo {
```
Comment (adairrr) [inline cw-orch/src/daemon/networks/migaloo.rs:20]: We should include some doc comments as to where this information came from, similar to other networks: 
https://github.com/AbstractSDK/cw-orchestrator/blob/e4bf3c350bfdf2554d5bda76a2540d26e7cd7a5d/cw-orch/src/daemon/networks/injective.rs#L10-L22

File: cw-orch/src/daemon/networks/migaloo.rs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    kind: ChainKind::Testnet,
+    chain_id: "narwhal-1",
```
Comment (adairrr) [inline cw-orch/src/daemon/networks/migaloo.rs:22]: qq - the const is `NARWHAL_2` but the `chain_id` is `narwhal-1`?

**Review by adairrr — CHANGES_REQUESTED** [review]
And finally - would we be able to add a quick doc for the supported chain for the docs?

https://github.com/AbstractSDK/cw-orchestrator/blob/972a0006dddab72ec5a6669796a5d5929d008d41/docs/src/chains/kujira.md#L1-L18

File: cw-orch/src/daemon/networks/migaloo.rs (hunk 2 above)
Comment (kakucodes) [reply cw-orch/src/daemon/networks/migaloo.rs:24]: good catch

File: cw-orch/src/daemon/networks/migaloo.rs (hunk 1 above)
Comment (kakucodes) [reply cw-orch/src/daemon/networks/migaloo.rs]: ✅

**Comment by kakucodes** [comment]
@adairrr thanks for the review. ready for re-review as all the points have been addressed.

File: cw-orch/src/daemon/networks/mod.rs [hunk 3]
```diff
@@ -16,6 +17,7 @@ pub use archway::{ARCHWAY_1, CONSTANTINE_3};
 pub use injective::{INJECTIVE_1, INJECTIVE_888};
 pub use juno::{JUNO_1, LOCAL_JUNO, UNI_6};
 pub use kujira::HARPOON_4;
+pub use migaloo::{LOCAL_MIGALOO, NARWHAL_1};
```
Comment (adairrr) [inline cw-orch/src/daemon/networks/mod.rs:20]:
```suggestion
pub use migaloo::{LOCAL_MIGALOO, NARWHAL_1, MIGALOO_1};
```

File: cw-orch/src/daemon/networks/mod.rs [hunk 4]
```diff
@@ -42,11 +44,13 @@ pub fn parse_network(net_id: &str) -> ChainInfo {
         CONSTANTINE_3,
         ARCHWAY_1,
         PION_1,
+        NARWHAL_1,
         NEUTRON_1,
         INJECTIVE_1,
         HARPOON_4,
         OSMO_5,
         LOCAL_OSMO,
+        LOCAL_MIGALOO,
```
Comment (adairrr) [inline cw-orch/src/daemon/networks/mod.rs:53]:
```suggestion
        LOCAL_MIGALOO,
        MIGALOO_1,
```

**Review by adairrr — CHANGES_REQUESTED** [review]

**Comment by kakucodes** [comment]
@adairrr tyty. one more re-review. we've got it this time

**Pull request merged**